README.md

```diff
@@ -3,28 +3,72 @@
 The Ansible Collection for Scale Computing HyperCore ([HyperCore](https://www.scalecomputing.com/sc-hypercore))
 a variety of Ansible content to help automate the management of Scale Computing HyperCore products.
 
+Documentation is available at https://scalecomputing.github.io/HyperCoreAnsibleCollection/.
 
+# Requirements
 <!--start requires_ansible-->
-## Ansible version compatibility
+## Ansible version requirements
 
 This collection has been tested against following Ansible versions: **>=2.12**.
 
 The collection should work with any Ansible version **>=2.9.10**,
 but this is not granted.
 <!--end requires_ansible-->
 
-## Python version compatibility
+## Python version requirements
 
 This collection requires Python 3.8 or greater.
 
-## HyperCore cluster version compatibility
+## HyperCore cluster version requirements
 
 This collection has been tested against following HyperCore cluster versions:
 - v9.1.14.208456
 - v9.2.13.211102
 - v9.3.5.212852
 - v9.4.0.213268
 
+# Installation
+
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
```

Comment:
Maybe something like this:
"To use this collection, first install it using the Ansible Galaxy command-line tool:"

Reply:
I will leave this as is - I verbatim copied from RH template.